docs/Contribution.md

```diff
@@ -0,0 +1,104 @@
+# Contribution Guide
+
+Thank you for your interest in contributing to Mobile App Control Center, Systers! There are always bugs to file; bugs to fix in the code; improvements to be made to the documentation; and more.
+
+
+The instructions given below are for software developers who want to work on [our current code](https://github.com/systers/macc).
+
+## Git workflow
```

Comment:
Git workflow should be explained first I suppose?
Then what to work on, and the guidelines to follow (coding styles)